internal/core/workers/operation_execution_worker/operation_execution_worker.go

```diff
@@ -154,7 +154,7 @@ func (w *OperationExecutionWorker) Start(ctx context.Context) error {
 		if err != nil {
 			loopLogger.Error("failed to finish operation", zap.Error(err))
 		}
-		err = w.operationManager.RevokeLease(operationContext, op)
+		err = w.operationManager.RevokeLease(ctx, op)
```

Comment:
@caiordjesus @lyracampos  We can not use the operation context for this method since the worker calls this method and is not attached to the operation execution. Using operation context here causes an error in which the lease is not revoked if the operation is canceled, it fails.